golem-base-indexer/golem-base-indexer-migration/src/m20250909_062255_create_golem_base_timeseries.rs

```diff
@@ -0,0 +1,49 @@
+use sea_orm_migration::prelude::*;
+
+#[derive(DeriveMigrationName)]
+pub struct Migration;
+
+#[async_trait::async_trait]
+impl MigrationTrait for Migration {
+    async fn up(&self, manager: &SchemaManager) -> Result<(), DbErr> {
+        let sql = r#"
+CREATE MATERIALIZED VIEW golem_base_timeseries AS
+WITH hourly_changes AS (
```

Comment:
Right now this query is too expensive. It didn't complete in 15 minutes on our kaolin instance.

We're reworking `golem_base_entity_history`, so for now lets not merge this, but wait and see if new golem_base_entity_history fixes the issue.